Update features.md

FlagHater opened the pull request:
It works!!

j10a1n15 reviewed (approved):
All good things come in three PR's

j10a1n15 commented on pages/features/list.md:
```diff
@@ -1,9 +1,20 @@
 
+### Mod Checker
+
+When enabled, Mod Checker automatically will check your mods list for outdated, suspicous, or unknown mods. These mods may not be on the verified mods list. If you believe a mod is being falsely flagging a mod, please report it in the PSS discord (``/pss``)
```
> If you believe a mod is being falsely flagging a mod
that sentence is weird

j10a1n15 commented on pages/features/list.md:
```diff
@@ -14,6 +25,10 @@ along with an option to select a random background.
 
+### Fun Fact on Main Menu
+
+Displays the daily fun fact on the PSS custom main menu. 
```
"displays **a** daily..." sounds better

j10a1n15 commented on pages/features/list.md:
```diff
@@ -119,13 +130,26 @@ party message and World War II (1939-1945) air raid siren in case the previous t
 
+### Healer Alert
+
+Sends a warning banner when enabled whenever a dungeon teammate is low on health. Togglable to change from a alert at 50% health to 25% health. 
```
I would remove "when enabled" in the first sentence
also its "a**n** alert lol

j10a1n15 commented on pages/features/list.md:
```diff
@@ -119,13 +130,26 @@ party message and World War II (1939-1945) air raid siren in case the previous t
 
+### Dungeon Snitcher
+
+Calls out players who may be slacking in dungons, using the Dungon Player Rater. Uses a configurable automatated message. 
```
dungon

FlagHater commented:
Lesson learned stop: stop writing things at 2 in the morning

j10a1n15 commented:
> Lesson learned stop: stop writing things at 2 in the morning

dungon

FlagHater commented:
> > Lesson learned stop: stop writing things at 2 in the morning
> 
> dungon

What if I told you that that wasn't related to me being wake at 2 in the morning but I actually did not know how to spell dungeons...

j10a1n15 commented:
> > > Lesson learned stop: stop writing things at 2 in the morning
> > 
> > 
> > dungon
> 
> What if I told you that that wasn't related to me being wake at 2 in the morning but I actually did not know how to spell dungeons...

lmaoo

FlagHater commented:
I think I fixed everything

j10a1n15 commented on pages/features/list.md:
```diff
@@ -1,9 +1,20 @@
 
+### Mod Checker
+
+When enabled, Mod Checker automatically will check your mods list for outdated, suspicous, or unknown mods. These mods may not be on the list of varified mods, a mod is being falsely flagged, please report it in the PSS discord (``/pss``)
```
varified
**if** a mod is being...

Su386yt commented:
Bump

Su386yt commented on pages/features/list.md:
```diff
@@ -1,9 +1,20 @@
+### Mod Checker
+
+When enabled, Mod Checker automatically will check your mods list for outdated, suspicous, or unknown mods. These mods may not be on the list of varified mods, a mod is being falsely flagged, please report it in the PSS discord (``/pss``)
+<img width="750" alt="18b5977a-c6b6-4f83-80bf-61b6a1297946" src="https://github.com/FlagHater/partly-sane-skies-FlagHater/assets/106500506/24d3ee14-bc5e-42e7-8604-c2811216b457">
```
Put the images in the images folder

Su386yt commented on pages/features/list.md:
```diff
@@ -119,13 +130,26 @@ party message and World War II (1939-1945) air raid siren in case the previous t
+
+Calls out players who may be slacking in dungeons, using the Dungeon Player Rater. Uses a configurable automatated message. 
+
+![3c9449dd-b694-44d2-93e7-9d846b109912](https://github.com/FlagHater/partly-sane-skies-FlagHater/assets/106500506/f7eb2384-7c99-41ef-827b-90fd4dbb937b)
```
Same here

Su386yt commented on pages/features/list.md:
```diff
@@ -14,6 +25,10 @@ along with an option to select a random background.
 
 ![Custom Main Menu](/images/custom_main_menu.png)
```
This is an example of how to put images in the image folder

Su386yt reviewed (changes requested):
Add all features to readme
Maybe add some more screenshots
Missing Features:
[Dungeon Snitcher](https://github.com/PartlySaneStudios/partly-sane-skies/releases/tag/beta-v0.5.3)
[Treecap Cooldown](https://github.com/PartlySaneStudios/partly-sane-skies/releases/tag/beta-v0.5.3)
[Coins to cookies](https://github.com/PartlySaneStudios/partly-sane-skies/releases/tag/beta-v0.5)

Su386yt commented on pages/features/list.md:
```diff
@@ -1,9 +1,20 @@
 
+### Mod Checker
+
+When enabled, Mod Checker automatically will check your mods list for outdated, suspicous, or unknown mods. These mods may not be on the list of verified mods, and if you believe a mod is being falsely flagged, please report it in the PSS discord (``/pss``)
```
And command is ``/pssdiscord``

FlagHater replied:
> sentance

Su386yt replied:
If you want to talk about typos...

FlagHater replied:
I know but this isn't at 2 in the morning this is midday

Su386yt replied:
the difference is im stupid

FlagHater replied:
I totally don't make typos every hour of the day

j10a1n15 commented on pages/features/list.md:
```diff
@@ -1,9 +1,20 @@
+
+### Privacy
+
+This toggle blocks other mods wanting to send data to their servers. Currently supported mods are Essentials & Dungeon Guide. If you know more mods that send that kind of data, feel free to report that in our discord server. This is a privacy feature, and is off by default.
```
> This is a privacy feature, and is off by default.

Feels very weird

FlagHater replied:
I just copied Su's description from the mod, I don't know how to rephrase that, unless something like ``This feature is meant as a privacy feature, but/and is off by default``

Su386yt replied:
It's not off by default

FlagHater replied:
Someone spreading misinformation in releases?

Su386yt replied:
Where does it say that
<img width="553" alt="image" src="https://github.com/PartlySaneStudios/partly-sane-skies/assets/83100266/a492c175-ea76-46f8-88a8-5fafc2066e4f">

Su386yt replied:
<img width="633" alt="image" src="https://github.com/PartlySaneStudios/partly-sane-skies/assets/83100266/9bb6db2d-f472-4db6-9467-484ae4aa284d">
In disc as well

FlagHater replied:
Oh wait it wasn't releases it was in features I copied it from, I think you had added it and so I just copied that

j10a1n15 replied:
Just remove that line, ~~also wth su you didnt do stuff with the privacy mode~~

j10a1n15 commented:
Also while youre at it, you could also alr add the readme stuff for my two prs #305 and #304 <3

Su386yt commented:
> Also while youre at it, you could also alr add the readme stuff for my two prs #305 and #304 <3

Poor flag

Su386yt commented:
Bump

j10a1n15 commented:
Bump

Su386yt commented:
im gonna close this soon lol for inactivity

j10a1n15 commented:
hiii :3

j10a1n15 commented:
Holy shit

j10a1n15 commented on pages/features/list.md:
```diff
@@ -183,6 +183,11 @@ Shows which items are the most profitable for the copper cost. (Which ones give
+### Treecap Cooldown
+
+Dispalys a cooldown bar inspired by the 1.9+ attack indicator. Has Monkey pet support.
+[(Cooldown showcase)](https://github.com/PartlySaneStudios/partly-sane-skies/assets/83100266/69fdead9-a682-4fc9-9830-496443c162e5) 
```
Hello Hi please put the image in /images/

j10a1n15 commented on pages/features/list.md:
```diff
@@ -9,7 +9,7 @@ General and Miscellaneous features.
 
 When enabled, Mod Checker automatically will check your mods list for outdated, suspicous, or unknown mods. These mods may not be on the list of verified mods. If you believe a mod is being falsely flagged, please report it in the PSS discord (``/pssdiscord``)
-<img width="750" alt="18b5977a-c6b6-4f83-80bf-61b6a1297946" src="https://github.com/FlagHater/partly-sane-skies-FlagHater/assets/106500506/24d3ee14-bc5e-42e7-8604-c2811216b457">
+![image](https://github.com/PartlySaneStudios/partly-sane-skies/assets/83100266/601f2344-ed39-47c9-bd28-e6da2c09a639)
```
Hello Hi thats not /images/

FlagHater replied:
Don't know how to do that

j10a1n15 replied:
You paste that file in there

Su386yt replied:
You have to have a cloned repo on your computer. You can't use the GitHub online editor

j10a1n15 commented on pages/features/list.md:
```diff
@@ -130,6 +135,10 @@ party message and World War II (1939-1945) air raid siren in case the previous t
 
 Automatically parties everyone in a permanent dungeon party. Using ``/pp`` or ``/permparty``, you have the option to create, party, delete, and add and remove members from your permanent party. You can even add different parties such as an f6Party or a jujuNonCarry party.
 
+### Pearl Refill
```
Alr exist, please revert that commit

j10a1n15 commented on pages/features/list.md:
```diff
@@ -51,6 +51,11 @@ When switching to a new location region on SkyBlock, an MMO RPG style banner wil
 
 ![Location Banner](/images/location_banner.png)
 
+### System Tray Notifications
```
Alr exist, please revert that commit

j10a1n15 commented on pages/features/list.md:
```diff
@@ -144,7 +144,7 @@ In a perfectly balanced 5-player party, each player should get 20%.
 Calls out players who may be slacking in dungeons, using the Dungeon Player Rater. Uses a configurable automatated message. 
 
-![image](https://github.com/PartlySaneStudios/partly-sane-skies/assets/83100266/5e1fa3f2-699d-4812-b757-867bcd9f4da7)
+![image](/images/dungeon_snicher.png)
```
This one seems to be broken, maybe it was a diff one

Su386yt replied:
Typo should say snitched

j10a1n15 commented on pages/features/list.md:
```diff
@@ -186,7 +186,7 @@ Shows which items are the most profitable for the copper cost. (Which ones give
 
 Dispalys a cooldown bar inspired by the 1.9+ attack indicator. Has Monkey pet support.
-[(Cooldown showcase)](https://github.com/PartlySaneStudios/partly-sane-skies/assets/83100266/69fdead9-a682-4fc9-9830-496443c162e5) 
+[(Cooldown showcase)](/images/Treecap_Cooldown.mp4) 
```
Videos apaprently dont get displayed idk

FlagHater replied:
Should I remove it then?

Su386yt replied:
Up to you, we could just have a link

FlagHater replied:
Yay a link! I'll try to do that instead then

Su386yt replied:
I meant just leave it rn

Su386yt replied:
But I think you're missing an exclamation

j10a1n15 replied:
> But I think you're missing an exclamation

wait that could be it (why it didnt get displayed)

Su386yt replied:
> Yay I can finally have a link! I'll try to do that instead then

When you dont have an exclamation or it doesn't point to something displayable, it turns into a hyperlink to the file, which in this case, is a video. I don't mean make the video a link.

Su386yt commented on images/Treecap_Cooldown.mp4:
Lowercase pls

j10a1n15 commented:
You also need to lowercase the filenames no?

FlagHater commented:
Revamped README, added Github, Discord, and Modrinth logo. Made /features/custom_main_menu.md image loader easier to customize. Made README images easier to customize and change in size. Fixed scaling inconsistency in some rectangular features images. Added Modrinth and Github logos next to their respective download links. Made Modrinth and Github logo hyperlinks to the PSS download pages on their respective websites Made Discord logo a hyperlink for the PSS discord. Added Unicode Characters to the drop-down menus to make it more ✨stylish✨ and to spruce it up a bit. Removed some features the newly re-named Top-Features as they were unnecessary.

j10a1n15 reviewed (changes requested):
Ah sorry forgot that existed, can you add zhe PearlRefill Keybind (Default P)?

FlagHater commented:
> Ah sorry forgot that existed, can you add zhe PearlRefill Keybind (Default P)?

no

FlagHater commented:
(In 1-2 business weeks sure)

j10a1n15 commented:
Flag please

j10a1n15 commented:
(In the keybind section btw)

Su386yt commented on README.md:
```diff
@@ -213,12 +216,19 @@ kicking, party transfer buttons, etc. Included in the Party Manager GUI.
+                                    
 ### Pearl Refill
 
 Do you use pearls for your Dungeon runs and want to refill them backup to 16? With the Pearlrefill command you can automatically refill it back up to 16 from any other Stacksize, there is also a Keybind (Default P) and an option to enable auto refill at Dungeon start (use at your own risk)
```
> Ah sorry forgot that existed, can you add zhe PearlRefill Keybind (Default P)?
@j10a1n15 

are you stoopid

Su386yt replied:
i did improve it

j10a1n15 replied:
Why do we have two pearl refill entries

Su386yt replied:
<img width="815" alt="image" src="https://github.com/PartlySaneStudios/partly-sane-skies/assets/83100266/aac69544-86b7-4a3e-8454-bd3c44644a9f">

wdym

Su386yt commented on README.md:
```diff
@@ -213,11 +198,16 @@ kicking, party transfer buttons, etc. Included in the Party Manager GUI.
 ### Pearl Refill
 
-Do you use pearls for your Dungeon runs and want to refill them backup to 16? With the Pearlrefill command you can automatically refill it back up to 16 from any other Stacksize, there is also a Keybind (Default P) and an option to enable auto refill at Dungeon start (use at your own risk)
+Do you use pearls for your Dungeon runs and want to refill them backup to 16? With the Pearl Refill command you can automatically refill it back up to 16 from any other stack size, there is also a hot key (Default ``P``) and an option to enable auto refill at Dungeon start (use at your own risk)
```
> Ah sorry forgot that existed, can you add zhe PearlRefill Keybind (Default P)?
@j10a1n15 

are you stoopid

j10a1n15 replied:
I meant in the keybinds section

Su386yt commented:
Fixed weird spacing you had and fixed typos

Su386yt reviewed (approved):
lgtm

j10a1n15 commented on README.md:
```diff
@@ -213,11 +198,16 @@ kicking, party transfer buttons, etc. Included in the Party Manager GUI.
 ### Pearl Refill
 
-Do you use pearls for your Dungeon runs and want to refill them backup to 16? With the Pearlrefill command you can automatically refill it back up to 16 from any other Stacksize, there is also a Keybind (Default P) and an option to enable auto refill at Dungeon start (use at your own risk)
+Do you use pearls for your Dungeon runs and want to refill them backup to 16? With the Pearl Refill command you can automatically refill it back up to 16 from any other stack size, there is also a hot key (Default ``P``) and an option to enable auto refill at Dungeon start (use at your own risk)
```
![image](https://github.com/PartlySaneStudios/partly-sane-skies/assets/45315647/2e8c4d0d-2162-4571-af16-340256819eee)
